tool/planet/etcd.go

```diff
@@ -170,6 +170,7 @@ func etcdEnable(upgradeService bool) error {
 	defer cancel()
 
 	if !upgradeService {
+		restartEtcdClients(ctx)
```

Comment:
To clarify this change - when the etcd service (not upgrade service!) is re-enabled (being the last step), the clients are restarted?

Reply:
correct. The upgrade service is a temp cluster that external clients can't reach, that is used to restore the backups. Then it get's restarted back on the regular ports, and this is when we trigger the restart.